webapp/components/DeleteData/DeleteData.vue

```diff
@@ -52,8 +61,19 @@ export default {
       deleteContributions: false,
       deleteComments: false,
       enableDeletionValue: null,
+      currentUserCount: [],
```

Comment:
This makes sense here to have this in a separate variable in my eyes.

I would name it with plural `currentUserCounts`.
And this is initial no array. It’s an object:
`currentUserCounts: {},`